src/index.js

```diff
@@ -177,6 +181,16 @@ if ( !! window.wcCalypsoBridge.isEcommercePlan ) {
 					}
 					return page;
 				} );
+
+				pages.push( {
+					container: Plugins,
+					path: '/plugins-upgrade',
+					breadcrumbs: [ 'Plugins', 'Upgrade' ],
```

Comment:
I'm unsure if we need `Upgrade`, but I think we need the menu to display `Plugins`

<img width="195" alt="image" src="https://github.com/Automattic/wc-calypso-bridge/assets/3747241/3256139c-0b2a-4be2-af0c-14ac662162f2">

Reply:
Thank you for checking typos 🙇 

I'll update the breadcrumb to `Plugins` 👍

Reply:
@ilyasfoo Updated in https://github.com/Automattic/wc-calypso-bridge/pull/1278/commits/ec4a373bb96500dab748c5bf61ae01d604c21308